src/commands/build/category.txt

```diff
@@ -0,0 +1 @@
+TODO: Build command category description
```

Comment:
@lukekim,@mhr3, @siminapasat Could you, please, clarify what category description is expected for "build" command category?